test/restore-object.js

```diff
@@ -0,0 +1,85 @@
+"use strict";
+/* eslint-disable no-empty-function */
+
+var referee = require("@sinonjs/referee");
+var sinonSpy = require("../lib/sinon/spy");
+var sinonStub = require("../lib/sinon/stub");
+var restoreObject = require("../lib/sinon/restore-object.js");
+var assert = referee.assert;
+var refute = referee.refute;
+
+describe("restore-object", function() {
+    it("is defined", function() {
+        assert.isFunction(restoreObject);
+    });
+
+    it("works on empty objects", function() {
+        refute.exception(function() {
+            restoreObject({});
+        });
+    });
+
+    it("quietly ignores falsy input", function() {
+        refute.exception(function() {
+            restoreObject(false);
```

Comment:
Same as comment above. This should throw a meaningful exception. The user is doing something dodgy. The argument is clearly not a result of calling `sinon.stub(...)`.

Reply:
Same as my comment above.